eth_tester/backends/mock/factory.py

```diff
@@ -236,8 +236,8 @@ def make_receipt(transaction, block, transaction_index, overrides=None):
 EMPTY_UNCLE_HASH = b'\x1d\xccM\xe8\xde\xc7]z\xab\x85\xb5g\xb6\xcc\xd4\x1a\xd3\x12E\x1b\x94\x8at\x13\xf0\xa1B\xfd@\xd4\x93G'  # noqa: E501
 
 
-def make_genesis_block():
-    return {
+def make_genesis_block(overrides=None):
+    genesis_block = {
```

Comment:
I'd suggest using the name `default_genesis_block`.  This will work well in conjunction with removing the mutation via `update` later in this function.